content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -125,42 +133,136 @@ using respective paltfroms for their repositories.
+          } else {
+            bat './scripts/test.bat'
+          }
+        }
+      }
+      post {
+        success {
+          junit 'target/surefire-reports/*.xml'
+        }
+        always {
+          findbugs pattern: 'target/*.xml'
+        }
+      }
+    }
+  }
+}
+----
+==== Pipeline view in Jenkins instance
+
+image:/images/post-images/gsoc-simple-pull-request-plugin/pipeline-view.png[title="Jenkinsfile.yaml", role="center"]
 
-image:/images/post-images/gsoc-simple-pull-request-plugin/YAML.png[title="Jenkinsfile.yaml", role="center"]
-
-Below pipeline code will be generated using above yaml file.
-
-image:/images/post-images/gsoc-simple-pull-request-plugin/Pipeline.png[title="Declarative Pipeline", role="center"]
 
 === Coding Phase 2 plans
 
 . Decide a proper YAML format to use for Jenkinsfile.yaml
-. Create Step Configurator for SPRP plugin. https://issues.jenkins-ci.org/browse/JENKINS-51637[JENKINS-51637].
+. Create Step Configurator for SPRP plugin. https://issues.Jenkins-ci.org/browse/Jenkins-51637[Jenkins-51637].
 This will enable users to use Pipeline steps in Jenkinsfile.yaml.
-. Geting rid of manual tab generation in Pipeline SnippetGenerator class.
+. Getting rid of manual tab generation in Pipeline SnippetGenerator class.
 . Write tests for the plugin.
 
+https://issues.Jenkins-ci.org/browse/Jenkins-51809[Jira Epic]
+
 
 === How to reach me
 
 * Email: gautamabhishek46@gmail.com
-* Gitter room: link:https://gitter.im/jenkinsci/simple-pull-request-job-plugin[]
+* Gitter room: link:https://gitter.im/Jenkinsci/simple-pull-request-job-plugin[]
 
 === References
 
 * link:https://docs.google.com/document/d/1cuC0AvQG3e4GCjIoCwK3J0tcJVAz1eNDKV8d_zXxlO8/edit[Initial proposal of the project]
-* link:https://github.com/jenkinsci/simple-pull-request-job-plugin[Project repository]
-* link:https://jenkins.io/projects/gsoc/2018/simple-pull-request-job-plugin/[Project page]
-* link:https://gitter.im/jenkinsci/simple-pull-request-job-plugin?utm_source=share-link&utm_medium=link&utm_campaign=share-link[Gitter chat]
-* link:https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20JENKINS%20AND%20component%20%3D%20simple-pull-request-job-plugin[Bug Tracker]
+* link:https://github.com/Jenkinsci/simple-pull-request-job-plugin[Project repository]
+* link:https://Jenkins.io/projects/gsoc/2018/simple-pull-request-job-plugin/[Project page]
+* link:https://gitter.im/Jenkinsci/simple-pull-request-job-plugin?utm_source=share-link&utm_medium=link&utm_campaign=share-link[Gitter chat]
+* link:https://issues.Jenkins-ci.org/issues/?jql=project%20%3D%20Jenkins%20AND%20component%20%3D%20simple-pull-request-job-plugin[Bug Tracker]
+* https://github.com/gautamabhishek46/dummy[Demo Repository]
+* https://www.youtube.com/watch?v=qWHM8S0fzUw[14 June Phase 1 evaluation presentation vedio]
+* https://docs.google.com/presentation/d/1R63XK4Nmd5EBmMdF2rkevV1iZxjTom19XFHdgjY9qeA/edit?usp=sharing[14 June Phase 1 evaluation PPT]
```

Comment:
I would reformat the link text to `Phase 1 Presentation Slides` and put the date after in parenthesis `(June 14, 2018)`.